src/components/RecipeDetails/RecipeDetails.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
+import Breadcrumb from "../Breadcrumb/Breadcrumb";
+import { BsFillTagFill } from "react-icons/bs";
+import RecipeNutrition from "./RecipesNutrition";
+import RecipeIngredients from "./RecipeIngredients";
+
+const ProductDetails = ({ uri }) => {
+  const [recipe, setRecipe] = useState({});
```

Comment:
An empty object is truthy, so perhaps `null` would be a better default value for a recipe.